api_v3/lib/KalturaFrontController.php

```diff
@@ -216,6 +216,15 @@ public function handleMultiRequest()
 		// process the requests
 		$results = array();
 		kCurrentContext::$multiRequest_index = 0;
+		if (isset($commonParams['ks']))
+		{
+			kEventsManager::enableMultiDeferredEvents(true);
+		}
+		else
+		{
+			kEventsManager::enableMultiDeferredEvents(false);
+		}
+
 		for($i = $requestStartIndex; $i <= $requestEndIndex; $i++)
 		{
 			$currentParams = $listOfRequests[$i];  
```

Comment:
If $currentParams contains ks that is different than the one in $commonParams then the multi-deferred should be disabled.